cmd/gossamer/configcmd_test.go

```diff
@@ -333,11 +435,17 @@ func TestCommands(t *testing.T) {
 		{"config", tempFile.Name(), "TOML configuration file"},
 	}
 
+	genesispath, err := filepath.Abs("../../genesis.json")
```

Comment:
Please use `createTempGenesisFile` for all calls like this

Reply:
fixed